### ISSUE TITLE
[1.20.1] Blocks bugged and not breaking in Crypt in Vethea Dimension

### ISSUE THREAD

**Issue by Spookygalore** [DivineRPG/DivineRPG#660]
They break on creative mode but not survival which is why some blocks are broken, chest also work on survival to open but I cannot mine or break them

Video: https://drive.google.com/file/d/1BIlouJ1YgcNTURzmSbNJsawTGRREBve-/view?usp=drivesdk

**Comment by velocityraptor-real** [comment]
Intended Feature. use dissipators to get rid of heavy air

**Comment by Magomed2213** [comment]
The same problem, but it has also spread to the overworld, what to do and where to find the diffuser? 


UPD: I found the answer for the 
 Overworld, for some reason the regime changed, survival for adventure. But I still haven't figured out the heavy air. Where and how to find a dissipators, how to remove heavy air?

**Comment by Magomed2213** [comment]
@velocityraptor-real

**Comment by velocityraptor-real** [comment]
The heavy air gives you a potion effect, which turns you temporarily into adventure mode. Once the effect is over though, you should be set back to survival gamemode. So you either managed to keep the effect or get rid of it without the effect updating and setting you back to survival mode. Did you perhaps die whilst in heavy air and respawned in the overworld?

**Comment by Magomed2213** [comment]
Yes, I died with the heavy air effect. Now I don't know how to remove this effect, I looked all over the Internet in search of an answer. help please 🥲

**Comment by velocityraptor-real** [comment]
do /effect clear, unless you do not have that effect on you anymore, then do /gamemode survival

**Comment by velocityraptor-real** [comment]
Bug reopened, as apparently the heavy air effect is not death safe. Will be fixed in a future update.

**Comment by Magomed2213** [comment]
I can't break blocks in the vesey dungeon, the dungeon was in the video that was posted above. Because of this effect, I can't break blocks in this dungeon, how to remove this effect in this particular dungeon. Also in the boss arena in the world of Vesey the same effect which is why I can not pass the bosses

**Comment by velocityraptor-real** [comment]
intended feature. you can remove heavy air via dissipators. this bug is open due to keeping the heavy air effect after death in vethea.

**Comment by Magomed2213** [comment]
where to find it dissipators

**Comment by velocityraptor-real** [comment]
you make them in an infusion table with a dissipator template which you can buy from the hunger.

**Comment by Magomed2213** [comment]
You mean this one ? 
https://drive.google.com/file/d/1cmU4JI3ZhT5LYsEs1YVGI6bq-2yNsc3u/view?usp=sharing

**Comment by velocityraptor-real** [comment]
yes

**Comment by Magomed2213** [comment]
How to use it to disperse heavy air, you need to throw it at the enemy or what to do?

**Comment by NicosaurusRex99** [comment]
throw it near the blocks you wish to interact with

**Comment by Magomed2213** [comment]
Unfortunately, it doesn't help, for some reason it doesn't come back, but just flies away and disappears

**Comment by velocityraptor-real** [comment]
The blocks it interacted with now transformed to normal air and will not give you the heavy air effect

**Comment by velocityraptor-real** [comment]
the better the tier of the dissipator, the bigger is the range of blocks around it that get changed

**Comment by Magomed2213** [comment]
Does it make any difference if I give it out using a creative or make it myself?

**Comment by velocityraptor-real** [comment]
the only difference is wether you cheated or not

**Comment by Magomed2213** [comment]
https://drive.google.com/file/d/1Sztt2ASs8Fob5kOKn2UOUEFqyCmo3VkO/view?usp=sharing 
it doesn't work for some reason, maybe I'm doing something wrong, specially took the most powerful weapon.

**Comment by velocityraptor-real** [comment]
what mod version are you using?

**Comment by Magomed2213** [comment]
1.10.7.5

**Comment by Rikiki007** [comment]
Hi, I do seems to have the same problem.
I crafted a Dissipator to use it in those crypth in Vethea to remove heavy air, but it does not work and I still have the Heavy air effect.
And it said suppose to Returns to the user but it doesnt comeback in the inventor and does the same for any disk.
I'm using the version 1.10.7.5

**Comment by velocityraptor-real** [comment]
in the recent updates, we fixed the effect behaviour, made removing it optional, by having the chests accessible without having to break a block. also the heavy air now generates particles, which makes it a lot more noticeable
